test/cli/autocorrect-useless-cast/autocorrect-useless-cast.rb

```diff
@@ -0,0 +1,17 @@
+# typed: strict
+
+T.cast("from", String)
+
+T.cast(
+  "multiline1",
+  String
+)
+
+T.cast(
+  "multiline2",
+  String)
+
+T.cast("multiline3",
+       String)
+
+T.cast(1.5, Float)
```

Comment:
ooc, any reason to keep this a CLI test vs using an `*.autocorrects.exp` file on a testdata test?

usually the only reason is if there's additional non-`setHeader` error context that you want to test is showing up correctly but I think that the `explainGot` helper is sufficiently well tested elsewhere.

Reply:
I did...not know about `*.autocorrects.exp`.

I'll leave it like this for now; maybe it'd be worth separating the CLI autocorrect tests into `*.autocorrects.exp` tests and keep the CLI tests for checking actual error text.

Reply:
eh, in a lot of cases it's annoying to have the Ruby source file in two places. In those cases, having a single CLI test is preferable to two tests that have the same Ruby source file.

But yeah, the `*.autocorrects.exp` format is more recent than many of the autocorrect CLI tests, and it's likely that some of them could be plainly converted to no longer be CLI tests.